src/data/ingestion.py

```
@@ -1,26 +1,73 @@
+        """
         try:
-            df = pd.read_excel(file_path)
-            df.columns = df.columns.str.strip()
+            # --- CRITICAL FIX HERE ---
+            # header=1 tells pandas to use the SECOND row as the column headers (since it's 0-indexed).
+            df = pd.read_excel(file_path, header=1)
+            # --- END OF CRITICAL FIX ---
+            
+            # Apply the column cleaning function for consistency
+            df = self._clean_column_names(df)
             return df
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)